NAMESPACE

```diff
@@ -0,0 +1,3 @@
+useDynLib(vivax, .registration=TRUE)
+exportPattern("^[[:alpha:]]+")
```

Comment:
Looks like this is not yet generated by roxygen (though there is at least the `@export` roxygen bits in the RcppAttributes generated code), and it would be better to move to explicit imports sooner rather than later.